cmd/abapEnvironmentCreateSystem.go

```diff
@@ -129,15 +141,53 @@ func generateManifestYAML(config *abapEnvironmentCreateSystemOptions) ([]byte, e
 	return manifestYAML, nil
 }
 
+func checkManifestParameters(config *abapEnvironmentCreateSystemOptions) (err error) {
+
+	//Only checks for correct parameter specification if CF Service "abap-oem" is selected
+	if config.CfService == "abap-oem" {
+
+		if config.AbapSystemParentSaasAppname == "" && config.AbapSystemParentServiceLabel == "" {
+			return errors.New("Both parameters AbapSystemParentServiceLabel and AbapSystemParentSaasAppname seem to be empty. Please specify either AbapSystemParentServiceLabel or AbapSystemParentSaasAppname depending on who created the oem-instance in the step configuration. For more information please refer to the step documentation")
+		}
+
+		if config.AbapSystemParentSaasAppname != "" {
+			const parentSaasAppnameSyntaxCheck = `[a-zA-Z0-9\-\_]+`
+			addonProductVersionMatch, _ := regexp.MatchString(parentSaasAppnameSyntaxCheck, config.AbapSystemParentSaasAppname)
+			if !addonProductVersionMatch {
+				return errors.New("It seems like you have incorrectly specified the AbapSystemParentSaasAppname step parameter. Please check that the parameters follows the respective syntax to specify the AbapSystemParentSaasAppname. For more information please refer to the step documentation")
+			}
+		}
+
+		if config.AbapSystemID != "" {
+			const systemIDSyntaxCheck = `[A-Z0-9]`
+			systemIDMatch, _ := regexp.MatchString(systemIDSyntaxCheck, config.AbapSystemID)
+			if !systemIDMatch {
+				return errors.New("It seems like you have incorrectly specified the AbapSystemID step parameter. Please check that the parameters follows the respective syntax to specify the AbapSystemID. For more information please refer to the step documentation")
+			}
+		}
+
+		if config.AbapSystemConsumerTenantLimit == 0 {
+			return errors.New("You have specified 0 tenants o be created in the system for the step parameter AbapSystemConsumerTenantLimit. Please check that you have set the parameter value correctly. For more information please refer to the step documentation")
```

Comment:
```suggestion
			return errors.New("You have specified 0 tenants to be created in the system for the step parameter AbapSystemConsumerTenantLimit. Please check that you have set the parameter value correctly. For more information please refer to the step documentation")
```

Reply:
I think we should not check the AbapSystemConsumerTenantLimit. For it seems this parameter is only "decoration". Parameter is optional an default value is actually "0"